dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -94,6 +94,22 @@ When a set of routes is created in various projects, the overall set of routes
 is available to the set of routers. Each router admits (or selects) routes from
 the set of routes. By default, all routers admit all routes.
 
+As of {product-title} 3.6, routers can accept IPV6 traffic from outside the
```

Comment:
i think it might be appropiate to add that this behaviour is by default and if there's an ipv6 interface available.